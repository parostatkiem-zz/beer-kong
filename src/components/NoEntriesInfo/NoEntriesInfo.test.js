import React from "react";
import { render } from "@testing-library/react";
import NoEntriesInfo from "./NoEntriesInfo";

describe("NoEntriesInfo", () => {
  it("Renders the default text", () => {
    const { container } = render(<NoEntriesInfo />);
    expect(container.textContent).toMatchSnapshot();
  });

  it("Renders a custom text", () => {
    const text = "test 123";
    const { container } = render(<NoEntriesInfo text={text} />);
    expect(container.textContent).toContain(text);
  });
});
