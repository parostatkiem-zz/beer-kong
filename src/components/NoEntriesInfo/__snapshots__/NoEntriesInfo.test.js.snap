// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoEntriesInfo Renders the default text 1`] = `" Ta lista nie ma jeszcze wpis√≥w"`;
